var/spack/repos/builtin/packages/spatial-index/package.py

```diff
@@ -16,6 +16,6 @@ class SpatialIndex(PythonPackage):
     depends_on("cmake")
     depends_on("boost")
 
-    version('0.0.1', tag='0.0.1', submodules=True)
+    version('0.0.1', tag='0.0.1-2', submodules=True)
```

Comment:
Really? We can't just call that `0.0.2`? It's just a patch release…

Reply:
It's still a deployed version, so we'd have to re deploy documentation with a new version, etc... This way it's obvious we're doing a quick fix to 0.0.1